src/Integration.Vsix/CFamily/FileConfig.cs

```diff
@@ -30,19 +30,36 @@ partial class CFamilyHelper
     {
         internal class FileConfig
         {
-            public static FileConfig TryGet(ProjectItem dteProjectItem, string absoluteFilePath)
+            public static FileConfig TryGet(ILogger logger, ProjectItem dteProjectItem, string absoluteFilePath)
             {
                 var vcProject = dteProjectItem.ContainingProject.Object as VCProject;
                 var file = dteProjectItem.Object as VCFile;
                 if (vcProject == null || file == null)
```

Comment:
I think it's worth logging this failure as well.

Reply:
Yes, I believe it is. But I don't really know in which case functionally we can reach this point with VCProject and VCFile are null.
Have you faced it before?

Reply:
Hmm, not sure. Perhaps a CMake file is not a VCProject? I'm not sure if it's here for defensive checks or if there was a real use case.

Reply:
No, I already checked that:
- If you mean CMake project then it will return with: `Not expecting to be called for files that are not in the solution` as they don't have a project
- if you mean Cmake files then they will return early as the `IsCFamilyDocument` will fail